### DISCUSSION TITLE
How do I run a Mathematica notebook from the Linix command line?

### DISCUSSION THREAD

**Discussion by friascogit** [sagemathinc/cocalc#7556] [General]
I have opened a Linux VM on CoCalc and installed the Wolfram Engine.
So I'm sitting in the Mathematica command line.
How do I run a Mathematica notebook? Where should I place this .nb file?

**Comment by williamstein** [comment]
We support running *Jupyter* notebooks with the Mathematica kernel.  We don't have a way to run Mathematica notebooks though.

**Reply by friascogit** [reply]
Thanks. Your how-to is great.
But converting Mathematica notebooks to Jupyter does not appear straightforward.
